### PR TITLE
Bios for Skonthebass

### PR DESCRIPTION
**Pull request by skonthebass**

### CODE REVIEW

**Comment by AGottipati** [comment]
Hi Steve, Going good :+1: ?

**Comment by t3hbeowulf** [comment]
Consider tapping 'Delete' when you see a login screen

**Comment by skonthebass** [comment]
It's going good @AGottipati   are you learning anything?

File: bios/skonthebass.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+This is my bios page
+
+I'd like to start this computer and get to the OS
+
+My favorite color is chicken
```
Comment (githubteacher) [inline bios/skonthebass.md:5]: How about the feathers?

File: bios/skonthebass.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
+This is my bios page
+
+I'd like to start this computer and get to the OS
```
Comment (wenyibing) [inline bios/skonthebass.md]: ask something here

**Comment by githubteacher** [comment]
@skonthebass love the bio :+1:

File: bios/skonthebass.md (hunk 1 above)
Comment (cnmontreal) [inline bios/skonthebass.md:5]: :100:

**Comment by skonthebass** [comment]
Closes #89

**Comment by skonthebass** [comment]
Fixes #89

**Pull request merged**